CHANGELOG.md

```diff
@@ -1,26 +1,71 @@
-## 1.5.5 (May 1, 2020)
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.5.7](https://github.com/simplepie/simplepie/compare/1.5.6...1.5.7) - 2021-12-19
+
+* Fix PHP8 crash due to insufficient isset test by @Alkarex in #670
+* gitignore tests by @Alkarex in #671
```

Comment:
IMO, the changelog should be curated and not contain inconsequential stuff like CI changes. Author names are probably too noisy for it.

Reply:
The format stated that Changelogs should be written for humans. For me it looks like the maintainers decided in the past to keep those changes (like changes to travis or composer) in the Changelog, so did I. But removing or rewriting those changes could be done in this or an next PR. This is up to the maintainers.

Reply:
Yeah, I interpret being written for humans as reducing noise since humans do not deal with it very well. Similarly, reader probably does not care who contributed each feature so I would suggest moving acknowledgements into a paragraph at the end – there we can thank the contributors while keeping the list of changes clean. But yeah, this is largely up to maintainers and their preferences.